cdap-ui/app/cdap/components/EntityListView/EntityListInfo/EntityListInfo.scss

```diff
@@ -42,6 +42,7 @@ $hovered-page-background-color: rgba(219, 219, 219, 1);
       font-size: 12px;
       display: inline-block;
       vertical-align: top;
+      transform: translateY(3px);
```

Comment:
This shouldn't be required. Just remove both `vertical-align` and `transform`? That would just align things right?
<img width="714" alt="screen shot 2017-03-14 at 11 25 13 am" src="https://cloud.githubusercontent.com/assets/1452845/23916379/31bfa812-08a9-11e7-892d-f925c2fe5e6b.png">

This is what I'm seeing after removing those two.